src/burger/BurgerDetails.java

```diff
@@ -1,4 +1,57 @@
 package burger;
 
 public class BurgerDetails {
+    private String nameOfBurger;
+    private String nameTopping;
+    private double priceOfBurger;
```

Comment:
variable name will be priceOfTheBurger

Reply:
code is updated according to your variable name priceOfTheBurger